.github/workflows/cross-compile.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: go build example/main.go
       - name: Run cross compilation
         # run in parallel on as many cores as are available on the machine
-        run: go tool dist list | xargs -I % -P "$(nproc)" .github/workflows/cross-compile.sh %
+        run: go tool dist list | grep -v "386" | xargs -I % -P "$(nproc)" .github/workflows/cross-compile.sh %
```

Comment:
Let's keep the compilation step. This doesn't cost us anything, now that the Go build cache is cached and restored.